src/torus.ts

```diff
@@ -70,10 +67,22 @@ class Torus {
 
   private legacyMetadataHost: string;
 
-  constructor({ enableOneKey = false, clientId, network, serverTimeOffset = 0, allowHost, legacyMetadataHost }: TorusCtorOptions) {
+  private _keyType: KeyType = "secp256k1";
+
+  constructor({
+    enableOneKey = false,
+    clientId,
+    network,
+    serverTimeOffset = 0,
+    allowHost,
+    legacyMetadataHost,
+    keyType = "secp256k1",
+  }: TorusCtorOptions) {
     if (!clientId) throw Error("Please provide a valid clientId in constructor");
     if (!network) throw Error("Please provide a valid network in constructor");
-    this.ec = new EC("secp256k1");
+
+    this._keyType = keyType;
```

Comment:
don't use underscore scheme.. private is already a correct modifier

Reply:
got it